site/docs/v1/tech/admin-guide/cli.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: doc
+title: FusionAuth CLI
+description: Learn about the FusionAuth admin CLI tool
+---
+
+The FusionAuth command line interface (CLI) tool allows you to manipulate FusionAuth from the command line.
+
+## Prerequisites
+
+The CLI tool requires node. It's tested with version 19 but should work with modern versions of node.
+
+## Installation
+
+You can install this using `npm`.
+
+```
+npm i -g @fusionauth/cli
+```
+
+
+Then you can run commands.
+
+```
+fusionauth theme:download -k <APIKEY> <themeid>
+
+# modify your theme
+
+fusionauth theme:upload -k <APIKEY> <themeid>
```

Comment:
I think there needs to be a little more description here of where the files need to be in order to upload.  I would assume that if I downloaded with the theme_id, they would be in a state to upload.  However the upload does not work for the theme id.  

Uploading theme acd1e588-236a-400c-bd35-16a38bbcaf46 from ./tpl/
Error uploading theme acd1e588-236a-400c-bd35-16a38bbcaf46:
ENOENT: no such file or directory, scandir './tpl/'

Reply:
Thanks @mark-robustelli , that's very helpful. Will see what we need to do to have it work transparently.

Reply:
Hmm. this works fine for me on the sandbox:

```
dan@mooreds cli % fusionauth theme:download -k 90d8fb62-6f13-47d4-8ef6-1c3e687883c6 -h https://sandbox.fusionauth.io cccc4d5d-4317-4c71-9e3c-2302c414380b
  _____          _                  _         _   _     
 |  ___|   _ ___(_) ___  _ __      / \  _   _| |_| |__  
 | |_ | | | / __| |/ _ \| '_ \    / _ \| | | | __| '_ \ 
 |  _|| |_| \__ \ | (_) | | | |  / ___ \ |_| | |_| | | |
 |_|   \__,_|___/_|\___/|_| |_| /_/   \_\__,_|\__|_| |_|
                                                        
Downloading theme cccc4d5d-4317-4c71-9e3c-2302c414380b to ./tpl/
Theme cccc4d5d-4317-4c71-9e3c-2302c414380b downloaded to ./tpl/
dan@mooreds cli % ls
tpl
dan@mooreds cli % fusionauth theme:upload -k 90d8fb62-6f13-47d4-8ef6-1c3e687883c6 -h https://sandbox.fusionauth.io cccc4d5d-4317-4c71-9e3c-2302c414380b  
  _____          _                  _         _   _     
 |  ___|   _ ___(_) ___  _ __      / \  _   _| |_| |__  
 | |_ | | | / __| |/ _ \| '_ \    / _ \| | | | __| '_ \ 
 |  _|| |_| \__ \ | (_) | | | |  / ___ \ |_| | |_| | | |
 |_|   \__,_|___/_|\___/|_| |_| /_/   \_\__,_|\__|_| |_|
                                                        
Uploading theme cccc4d5d-4317-4c71-9e3c-2302c414380b from ./tpl/
Theme cccc4d5d-4317-4c71-9e3c-2302c414380b was uploaded successfully
```

Can you show me what issues you are seeing?

Reply:
I'm not going to lie, just tried it again and it is working fine.  Very weird.  I spent a little time trying to re-create.  The only way I can is to delete the /tpl directory, so I must have done that without realizing.  Apologies.cl

![Screenshot 2023-07-27 at 5 11 28 PM](https://github.com/FusionAuth/fusionauth-site/assets/137117976/c5b41b53-674d-4f6c-882f-12be1ea4a891)

Reply:
Ah, that makes sense.